lib/docker.ts

```diff
@@ -1,12 +1,22 @@
+import * as Debug from "debug";
 import { fileSync } from "tmp";
 import {
-  ExtractedImage,
-  ExtractedKeyFiles,
-  extractImageKeyFiles,
+  ExtractAction,
+  ExtractCallback,
+  extractFromTar,
+  ExtractProductsByFilename,
 } from "./analyzer/image-extractor";
-import { execute } from "./sub-process";
+import { CmdOutput, execute } from "./sub-process";
 
-export { Docker, DockerOptions };
+export { Docker, DockerOptions, STATIC_SCAN_MAX_IMAGE_SIZE_IN_BYTES };
+
+const debug = Debug("snyk");
+
+const KB = 1024;
+const MB = KB * 1024;
+const GB = MB * 1024;
+
+const STATIC_SCAN_MAX_IMAGE_SIZE_IN_BYTES = 3 * GB;
```

Comment:
shouldn't we make it less? I think we should limit the static scan to ~40 secs

Reply:
3GB takes about 40 secs on my laptop but I would like to change the value to comply with the `MAX_IMAGE_SIZE_IN_BYTES` default value in `docker-registry-agent` which is set to 2GB